From 02fa15f343b7aab40bbd49cbec8924435adf23a7 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Sat, 22 Nov 2014 13:35:36 -0600
Subject: [PATCH 0543/1074] MLK-9910: ARM: dts: reduce the i.MX6sx AI board
 QSPI clock frequency

Reduced the QSPI clock from 53Mhz to 29Mhz according to spec requirement

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit ea5d6d29564c29b21d117056e9cbd1430199b4de)
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 849ab9b..59f305d 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -640,7 +640,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
-		spi-max-frequency = <53000000>;
+		spi-max-frequency = <29000000>;
 		reg = <0>;
 	};
 
@@ -648,7 +648,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
-		spi-max-frequency = <53000000>;
+		spi-max-frequency = <29000000>;
 		reg = <1>;
 	};
 };
-- 
1.7.5.4

